lib/network/modules/LayoutEngine.js

```diff
@@ -1741,40 +1762,12 @@ class LayoutEngine {
    */
   _sortNodeArray(nodeArray) {
```

Comment:
This doesn't seem very useful anymore, now that the sorting is handled by the strategy

Reply:
Well, only the test on length > 1 I guess. Removing `_sortNodeArray()` would make the `sort`'s in the strategy instances 3 lines longer each, and I wanted to avoid that to keep the sizes down for clarity.

Reply:
Is the micro-optimization of skipping sorts for 1 item lists worth it? I'd just drop this code completely unless it can be shown to have an noon negligible effect on performance.

Reply:
Agree, it's silly. In fact my argument is completely silly. Sorting moved to strategy instances.